python/paddle/dataset/common.py

```diff
@@ -20,9 +20,9 @@
 import sys
 import importlib
 import paddle.dataset
-import cPickle
+import pickle
 import glob
-import cPickle as pickle
+import pickle as pickle
```

Comment:
Why this file has two imports.

Reply:
Because the ```cPickle``` was imported twice in the original file